ui-tests/tst/software/aws/toolkits/jetbrains/uitests/fixtures/Preferences.kt

```diff
@@ -26,8 +26,6 @@ fun RemoteRobot.preferencesDialog(
     }
 }
 
-private fun RemoteRobot.preferencesTitle() = if (this.isMac()) "Preferences" else "Settings"
```

Comment:
Why move this? It felt preferences specific so thats why it was here

Reply:
It's used in other places as well, actually did not finish deduping all of the duplications of this

Reply:
I didn't read the name of the file, reverted